authentication/src/main/kotlin/com/uber/sdk2/auth/api/request/CrossApp.kt

```diff
@@ -15,14 +15,25 @@
  */
 package com.uber.sdk2.auth.api.request
 
-/** Provides different apps that could be used for authentication using SSO flow. */
-sealed class CrossApp {
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+/**
+ * Provides different apps that could be used for authentication using SSO flow.
+ *
+ * @param packages The list of packages that could be used for authentication.
+ */
+@Parcelize
+sealed class CrossApp(val packages: List<String>) : Parcelable {
   /** The Eats app. */
-  data object Eats : CrossApp()
+  data object Eats :
+    CrossApp(listOf("com.ubercab.eats", "com.ubercab.eats.exo", "com.ubercab.eats.internal"))
```

Comment:
should we extract them in some constants or stirngs.xml?

Reply:
I can extract them as constants in the same class